bridge/MixerManager.cpp

```diff
@@ -354,25 +373,28 @@ void MixerManager::engineMessageMixerRemoved(const EngineMessage::Message& messa
         auto& command = message.command.mixerRemoved;
 
         std::string mixerId(command.mixer->getId()); // copy id string to have it after EngineMixer is deleted
-        auto findResult = _mixers.find(mixerId);
-        if (findResult != _mixers.end())
+        auto mixer = eraseAndGet(_mixers, mixerId);
+        if (mixer)
         {
-            mixer = std::move(findResult->second);
             mixer->stopTransports(); // this will stop new packets from coming in
-            if (!mixer->waitForAllPendingJobs(1000))
+            if (mixer->hasPendingJobs())
```

Comment:
Consider renaming to 'pop' (though it's usually a queue method, it looks like it would fit here perfectly).

Reply:
do you mean on line `376` to replace `eraseAndGet`?